business_requirement_deliverable_cost/models/business.py

```diff
@@ -139,7 +161,7 @@ def action_button_update_estimation(self):
                 for resource in deliverable.resource_ids:
                     pricelist_id = resource._get_pricelist()
                     partner_id = resource._get_partner()
-                    resource.sale_price_unit = resource.product_id.lst_price
+                    resource.sale_price_unit = resource.product_id.list_price
```

Comment:
Yes it's ok. I will explain. `lst_price` is a related field to `list_price` so in the end is the same value but I have checked on the others BR modules and regards product we use `list_price` one so I rather use the same on all the modules.

`'list_price': fields.float('Sale Price', digits_compute=dp.get_precision('Product Price'), help="Base price to compute the customer price. Sometimes called the catalog price."),`

`'lst_price' : fields.related('list_price', type="float", string='Public Price', digits_compute=dp.get_precision('Product Price')),`